.gitignore

```diff
@@ -1,5 +1,5 @@
 # Documentation ignore
-docs/source/reference/generated
+docs/source/instrument-documentation/generated
```

Comment:
Should this be `code-documentation`?

Reply:
Yup, good catch

Reply:
Odd that the intellij refactor renamed it the first time but not the second time 🤔